server/package.json

```diff
@@ -27,13 +27,16 @@
     "moment": "^2.29.1",
     "mongodb": "^3.6.5",
     "mongoose": "^5.12.1",
+    "multer": "^1.4.2",
     "mysql2": "^1.7.0",
     "nodemailer": "^6.6.0",
     "path": "^0.12.7",
     "request": "^2.88.2",
     "request-promise": "^4.2.6",
     "sequelize": "^5.22.4",
     "sequelize-cli": "^5.5.1",
+    "socket-io": "^1.0.0",
```

Comment:
multiple socket-io includes

Reply:
Removed file from uploads and duplicate socket-io include from server/package.json